art/step/step.py

```diff
@@ -227,6 +229,12 @@ def __call__(
         """
         self.model.set_metric_calculator(metric_calculator)
         super().__call__(previous_states, datamodule, metric_calculator)
+        #save model to file
+        ModelSaver().save(self.model, self.get_step_id(), self.name)
+        del self.model
```

Comment:
Have you checked that this actually works and the memory is freed?

What if someone want to read step.model now?  Maybe it could be changed into model path or something?

Reply:
This won't solve the issue model will still be available from trainer